src/module.cpp

```diff
@@ -1896,7 +1902,12 @@ void Module::execPreprocessor(const char *infilename, llvm::raw_string_ostream *
     inst.setTarget(target);
     inst.createSourceManager(inst.getFileManager());
 
+#if ISPC_LLVM_VERSION <= ISPC_LLVM_9_0
     clang::FrontendInputFile inputFile(infilename, clang::InputKind::Unknown);
+#else // LLVM 10.0+
+    clang::FrontendInputFile inputFile(infilename, clang::Language::Unknown);
```

Comment:
Yes. enum Language was a part of class InputKind earlier. It got moved. InputKind is a class. What we are doing is this-
constexpr InputKind(Language L = Language::Unknown, Format F = Source,  bool PP = false)
This is wht this- https://clang.llvm.org/doxygen/LangStandard_8h_source.html header was added in trunk

Reply:
I think we just need to pass `InputKind()` instead of `InputKind`, which defaults to "unknown" InputKind (whatever it is, or going to be in the future).

Reply:
Yeah. That will work too. have changed to that.